nixos/roles/rabbitmq.nix

```diff
@@ -10,50 +10,51 @@ in
     mkRole = v: {
       enable = lib.mkEnableOption
         "Enable the Flying Circus RabbitMQ ${v} server role.";
-      supportsContainers = fclib.mkEnableContainerSupport;
     };
   in {
     flyingcircus.roles = {
-      rabbitmq = mkRole "3.8";
+      rabbitmq36_5 = {
```

Comment:
should we add the frozen suffix here or keep it as the old role?

Reply:
no, must be the same as the old role to allow upgrades without changing the role in the directory.